collectors/30/kafka_offset_collector.py

```diff
@@ -41,8 +42,29 @@ def format_kafka_consumer_offset_tsd_key(consumer_group, topic, partition, offse
 
 
 def get_partitions(zk, group, topic):
-    return zk.get_children(KafkaPaths.topic_offsets(group, topic))
-
+    try:
+        return map(int, zk.get_children(KafkaPaths.topic_offsets(group, topic)))
```

Comment:
Why is `map(int` in `get_partitions` but `map(str` not in `get_topics`? Can we be more consistent?